files/passwd

```diff
@@ -0,0 +1,14 @@
+root:x:0:0:root:/root:/bin/bash
```

Comment:
I am not sure if release-bot needs passwd file, since it is not interacting with the fedora.